src/FileSystem/SourceFileFilter.php

```diff
@@ -75,9 +73,9 @@ public function getFilters(): array
     }
 
     /**
-     * @param iterable<SplFileInfo&SourceFileData> $input
+     * @param iterable<object> $input
```

Comment:
What's the reason of this change?

Reply:
I couldn't really see a required type downstream. I assume SplFileInfo? But at the end couldn't see it so I dont see the point to couple it to SourceFileData

Reply:
From what I see in `SourceFileDataProvider` (in master), it is only `iterable<SourceFileData>`.

More strict - better IMO.

Reply:
I like `iterable<SplFileInfo&SourceFileData>` better because it comes up when you look up usage. Else you will be looking at SourceFileData and wonder why is there this funny `getSplFileInfo` and where does it go? Thank you for getting it back.